docs-ref-conceptual/manage-azure-subscriptions-azure-cli.md

```diff
@@ -41,5 +41,6 @@ If so, you can configure Azure CLI 2.0 to execute commands against a particular
    ```
 
 You can verify the change by running the `az account list --output table` command again.
+Note : You can also use SubscriptionId instead of Subscription Name to set default account.
```

Comment:
The `NOTE:` designation should be removed, and the wording should be changed to `You can use either the subscription ID or name to set the default account."

I would recommend that this line be moved to before the "You can verify...".